src/data_reader.py

```diff
@@ -98,14 +108,15 @@ def read_data(image_payload: list[dict]) -> str:
                 {
                 "type": "text",
                 "text": """Tell me the business names, totals and transaction dates. 
-                        Answer should be formatted like: [(name1, total1, date1), (name2, total2, date2), etc...].
-                        Only give me the list, nothing else.""",
+                        Answer should be formatted like: [('name1', 'total1', 'date1'), ('name2', 'total2', 'date2'), etc...].
+                        Only give me the list, nothing else.
+                        Give the total amount as numeric. No need to include the currency denomination.""",
                 },
                 *image_payload
             ],
             }
         ],
-        max_tokens=300,
+        max_tokens=1000,
```

Comment:
Any reason for upping the tokens?

Reply:
Oh yeah, I tried with a monthly bank statement, and it failed there. We need to find a sweet spot or probably batch the transactions so that they don't exceed max_tokens.